Core/src/org/sleuthkit/autopsy/textextractors/TikaTextExtractor.java

```diff
@@ -171,9 +169,29 @@
      *
      * @return Flag indicating if OCR is set to be used.
      */
-    private boolean ocrEnabled() {
-        return TESSERACT_PATH != null && tesseractOCREnabled
-                && PlatformUtil.isWindowsOS() == true && PlatformUtil.is64BitOS();
+    /**
+     * Method to indicate if OCR should be performed on this image file. Checks
+     * to see if the limited OCR setting is enabled. If it is it will also check
+     * that one of the limiting factors is true.
+     *
+     * @param file    The AbstractFile which OCR might be performed on.
+     * @param boolean The configuration setting which indicates if limited OCR
+     *                is enabled in Keyword Search.
+     *
+     * @return True if limited OCR is not enabled or the image is greater than
+     *         100KB in size or the image is a derived file.
+     */
+    @Override
+    public boolean willUseOCR() {
+        // If Tesseract has been installed and is set to be used through
+        // configuration, then ocr is enabled. OCR can only currently be run on 64
+        // bit Windows OS.
+        return TESSERACT_PATH != null
+                && tesseractOCREnabled
+                && PlatformUtil.isWindowsOS()
+                && PlatformUtil.is64BitOS()
+                // if limited OCR is not enabled or the image is greater than 100KB in size or the image is a derived file.
+                && content instanceof AbstractFile;
```

Comment:
Don't we need to refactor the MIME type check into this code to answer the question "will use OCR?" It seems like this implementation is only affirming that the text extractor might use OCR, not that it will use OCR.